Dockerfile

```diff
@@ -1,103 +1,137 @@
 # Dockerfile with:
 #   - Android build environment
 #   - python-for-android dependencies
+#
 # Build with:
-# docker build --tag=p4a .
+#     docker build --tag=p4a .
+#
 # Run with:
-# docker run p4a /bin/sh -c '. venv/bin/activate && p4a apk --help'
+#     docker run -it --rm p4a /bin/sh -c '. venv/bin/activate && p4a apk --help'
+#
 # Or for interactive shell:
-# docker run -it --rm p4a
+#     docker run -it --rm p4a
+#
+# Note:
+#     Use 'docker run' without '--rm' flag for keeping the container and use
+#     'docker commit <container hash> <new image>' to extend the original image
+
 FROM ubuntu:18.04
 
+ENV ANDROID_HOME="/opt/android"
 
-ENV USER="user"
-ENV HOME_DIR="/home/${USER}"
-ENV WORK_DIR="${HOME_DIR}" \
-    PATH="${HOME_DIR}/.local/bin:${PATH}"
-# get the latest version from https://developer.android.com/ndk/downloads/index.html
+RUN apt-get -y update -qq \
+    && apt-get -y install -qq --no-install-recommends curl unzip \
+    && apt-get -y autoremove \
+    && apt-get -y clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Minor can we replace the old fashion `apt-get` by simply `apt`?

Reply:
You can still use `autoremove` and stuff through `apt` just like we were doing back then with `apt-get`.
No specific reasons, it just feels too 2010 to use `apt-get` :joy: I like to stay up to date with the new "upstreamness" :wink: